src/sentry/static/sentry/app/views/settings/project/projectUserFeedback.jsx

```diff
@@ -0,0 +1,186 @@
+        setTimeout(
+          function() {
+            this._submitInProgress = false;
+            this.onSuccess();
+          }.bind(this),
+          500
+        );
+      };
+    };
+  }
+
+  componentWillUnmount() {
+    window.sentryEmbedCallback = null;
+  }
+
+  getEndpoints() {
+    let {orgId, projectId} = this.props.params;
+    return [
+      ['keyList', `/projects/${orgId}/${projectId}/keys/`],
+      ['project', `/projects/${orgId}/${projectId}/`],
+    ];
+  }
+
+  getInstructions() {
+    let dsn = this.state.keyList.length
+      ? this.state.keyList[0].dsn.public
+      : 'https://public@sentry.example.com/1';
+
+    return (
+      '<!-- Sentry JS SDK 2.1.+ required -->\n' +
+      '<script src="https://cdn.ravenjs.com/2.1.0/raven.min.js"></script>\n\n' +
+      '{% if request.sentry.id %}\n' +
+      '  <script>\n' +
+      '  Raven.showReportDialog({\n' +
+      '    // grab the eventId generated by the Sentry SDK\n' +
+      "    eventId: '{{ request.sentry.id }}',\n\n" +
+      '    // use the public DSN (dont include your secret!)\n' +
+      "    dsn: '" +
+      dsn +
+      "'\n" +
+      '  });\n' +
+      '  </script>\n' +
+      '{% endif %}\n'
+    );
+  }
+
+  getBrowserJSInstructions() {
+    let dsn = this.state.keyList.length
+      ? this.state.keyList[0].dsn.public
+      : 'https://public@sentry.example.com/1';
+
+    return (
+      '<!-- Sentry JS SDK 2.1.+ required -->\n' +
+      '<script src="https://cdn.ravenjs.com/2.1.0/raven.min.js"></script>\n\n' +
+      '<script>\n' +
+      '// configure the SDK as you normally would\n' +
+      "Raven.config('" +
+      dsn +
+      "').install();\n\n" +
+      '/**\n' +
+      ' * Report a routing error to Sentry and show a feedback dialog to\n' +
+      ' * the user.\n' +
+      ' * \n' +
+      ' * > try {\n' +
+      ' * >   renderRoute()\n' +
+      ' * > } catch (err) {\n' +
+      ' * >   handleRouteError(err);\n' +
+      ' * > }\n' +
+      ' */\n' +
+      'function handleRouteError(err) {\n' +
+      '  Raven.captureException(err);\n' +
+      '  Raven.showReportDialog();\n' +
+      '};\n' +
+      '</script>\n'
+    );
+  }
+
+  handleClick = () => {
+    Raven.showReportDialog({
+      // should never make it to the Sentry API, but just in case, use throwaway id
+      eventId: '00000000000000000000000000000000',
+    });
+  };
+
+  renderBody() {
+    let {orgId, projectId} = this.props.params;
+
+    return (
+      <div>
+        <SettingsPageHeader title={t('User Feedback')} />
+
+        <div className="alert alert-block alert-info">
+          {t(`Psst! This feature is still a work-in-progress. Thanks for being an early
+          adopter!`)}
+        </div>
+
+        <TextBlock>
+          {t(`Enabling User Feedback allows you to interact with your users on an
+          unprecedented level. Collect additional details about issues affecting them, and
+          more importantly reach out to them with resolutions.`)}
```

Comment:
I wonder what this looks like in Transifex (with all the superfluous whitespace). e.g. Is the whitespace part of the translation key? If it is, removing/adding whitespace in any form will break translations. (And if so, I wonder where else we make this mistake.)

Reply:
Yeah, whitespace seems to be part of the translation key